//this is an auto-generated bug report
//package name: eu.devunit.fb_client
//version: 1.2
//appro_time: 145
 FATAL EXCEPTION: AsyncTask #1
 java.lang.RuntimeException: An error occured while executing doInBackground()
 	at android.os.AsyncTask$3.done(AsyncTask.java:299)
 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:352)
 	at java.util.concurrent.FutureTask.setException(FutureTask.java:219)
 	at java.util.concurrent.FutureTask.run(FutureTask.java:239)
 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:230)
 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1080)
 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:573)
 	at java.lang.Thread.run(Thread.java:841)
 Caused by: java.lang.NullPointerException
 	at eu.devunit.fb_client.filebin.FilebinClient.getApiUri(FilebinClient.java:73)
 	at eu.devunit.fb_client.filebin.FilebinAsyncUploader.doInBackground(FilebinAsyncUploader.java:74)
 	at eu.devunit.fb_client.filebin.FilebinAsyncUploader.doInBackground(FilebinAsyncUploader.java:25)
 	at android.os.AsyncTask$2.call(AsyncTask.java:287)
 	at java.util.concurrent.FutureTask.run(FutureTask.java:234)
 	... 4 more
